R/aaa_lib_versions.R

```diff
@@ -0,0 +1 @@
+datePickerVersion <- "1.9.0"
```

Comment:
Can you update this from the `updateBootstrapDatepicker.R` script? Or at least modify that script to print out a message saying to update this value. (But just updating this file automatically doesn't seem like much more work.)